lepton/ldd_linux.go

```diff
@@ -71,8 +72,24 @@ func getSharedLibs(targetRoot string, path string) ([]string, error) {
 			if strings.HasPrefix(libpath, dir) {
 				libpath = libpath[len(dir)+1:]
 			}
+			if strings.Contains(text, "not found") {
+				notExistLib = append(notExistLib, text)
+			}
+			err = errors.New("")
 			deps = append(deps, strings.TrimSpace(libpath))
 		}
+
+		if len(notExistLib) != 0 {
+			errMessage := "Library: "
+			for i := range notExistLib {
+				errMessage += notExistLib[i]
+				if i+1 != len(notExistLib) {
+					errMessage += "\n"
+				}
+			}
+			return nil, errors.New(errMessage)
+		}
```

Comment:
this is good, however, a more verbose error msg would be good to have

along with this msg maybe say "Ops can't find a missing library:"

```
Library: libmissing.so => not found
```